metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -192,15 +192,10 @@ def gen_schema_containers(
         database: str,
         extra_properties: Optional[Dict[str, Any]] = None,
     ) -> Iterable[MetadataWorkUnit]:
-        database_container_key = gen_database_key(
-            database,
-            platform=self.platform,
-            platform_instance=self.config.platform_instance,
-            env=self.config.env,
-        )
+        database_container_key = self.get_database_container_key(database, schema)
```

Comment:
is this going to break existing URNs? 

if yes, let's place this in the release notes AT A MINIMUM?